src/brevitas/graph/equalize.py

```diff
@@ -222,27 +352,34 @@ def _extract_regions(graph_model: GraphModule):
         if node.op == 'call_module':
             module = get_module(graph_model, node.target)
             if isinstance(module, _supported_layers):
-                srcs, sinks = {node.target}, set()
+                srcs = {node.target: 0}
+                sinks = dict()
                 walk_region(graph_model, node, set(), srcs, sinks, walk_forward=True)
                 if sinks:
                     # each region should appear only once, so to make it hashable
                     # we convert srcs and sinks to ordered lists first, and then to tuples
```

Comment:
This comment doesn't seem to apply anymore given that they are not lists anymore